parsing/g_vernac.ml4

```diff
@@ -1145,33 +1132,33 @@ GEXTEND Gram
      | IDENT "Bind"; IDENT "Scope"; sc = IDENT; "with";
        refl = LIST1 class_rawexpr -> VernacBindScope (sc,refl)
 
-     | IDENT "Infix"; local = obsolete_locality;
+     | IDENT "Infix";
```

Comment:
To satisfy at the nitpicking tradition, I would delete the newline here.

Reply:
Sure, will fix, thanks!